flow360/component/simulation/meshing_param/edge_params.py

```diff
@@ -67,6 +68,8 @@ class SurfaceEdgeRefinement(Flow360BaseModel):
         ]
     ] = pd.Field(None, discriminator="type")
 
+    private_attribute_is_global_setting: bool = pd.Field(default=False, frozen=True)
```

Comment:
Why do we need this?
Front end need to know which ones are global settings. Front end use this to detect if it should 1. mask out non-global settings
2. disable "delete", "duplicate" action.
3. disable selecting entities for these global settings.

Why cannot we just see if entities is empty?
Because all user added refinements **initially** does not have `entities` and therefore all are considered global by this criterion. Therefore user cannot add entities to make it non-global...